src/components/layout/SimpleOrgLayout.tsx

```diff
@@ -0,0 +1,24 @@
+//TODO: Enable eslint rule and fix errors
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
+import { Content } from '@adobe/react-spectrum';
+import { useQuery } from 'react-query';
+
+import DefaultLayout from './DefaultLayout';
+import getOrg from '../../fetching/getOrg';
+
+interface OrgLayoutProps {
+    children: JSX.Element;
+    orgId: string;
+}
+
+const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
```

Comment:
The name here should be `SimpleOrgLayout` to reflect the public name of the component (i.e. the one that an importer will know).

Reply:
Sorry about this, was a bit too quick in the process of renaming them.